packages/FormGroup/src/FormGroup.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import Section from '@components/section';
+import readme from '../README.md';
+
+import FormGroup from './FormGroup';
+import Input from '@igloo-ui/input';
```

Comment:
it's a small detail but place the module import in first time, a the second place the intern module at last the local file.

**Example:** 
```
import React from 'react';
import Input from '@igloo-ui/input';
...

import Section from '@components/section';
...

import readme from '../README.md';
import FormGroup from './FormGroup';

```

Reply:
I think this should be fixed now, let me know if I missed something.